e2e/cypress/e2e/superadmin.cy.ts

```diff
@@ -42,49 +106,44 @@ describe('Superadmin', () => {
       .should('be.disabled');
   });
 
-  it('Should change the password of a existing user', () => {
-    cy.contains('workspace_admin')
+  it('should be possible to change privileges of existing workspace_admin to read-write', () => {
+    cy.contains(WorkspaceAdminName)
+      .should('exist')
       .click()
-      .get('[data-cy="change-password"]')
+      .get('[data-cy="workspace-role-rw"]')
+      .should('exist')
       .click()
-      .get('[formcontrolname="pw"]')
-      .type('newPassword')
-      .get('[type="submit"]')
+      .get('[data-cy="save"]')
       .click();
     logoutAdmin();
-    insertCredentials('workspace_admin', 'newPassword');
-    cy.get('[data-cy="login-admin"]')
+    loginAdmin(WorkspaceAdminName, WorkspaceAdminPw);
+    cy.contains('sample_workspace')
       .click();
-    cy.contains('Status: Angemeldet als "workspace_admin"')
-      .should('exist');
+    cy.url().should('eq', `${Cypress.config().baseUrl}/#/admin/1/files`);
+    cy.get('[data-cy="upload-files"]')
+      .should('be.enabled')
+      .get('[data-cy="delete-files"]')
+      .should('be.enabled');
   });
 
-  it('Should change privileges of existing admin to read-write', () => {
+  it('Should be possible to change the password of a existing workspaceadmin', () => {
     cy.contains('workspace_admin')
-      .should('exist')
       .click()
-      .get('[data-cy="workspace-role-ro"]')
-      .should('exist')
+      .get('[data-cy="change-password"]')
       .click()
-      .get('[data-cy="save"]')
+      .get('[formcontrolname="pw"]')
+      .type(newWorkspaceAdminPw)
+      .get('[type="submit"]')
       .click();
     logoutAdmin();
-    insertCredentials('workspace_admin', 'anotherPassword');
+    insertCredentials(WorkspaceAdminName, newWorkspaceAdminPw);
     cy.get('[data-cy="login-admin"]')
       .click();
-    cy.url().should('eq', `${Cypress.config().baseUrl}/#/r/admin-starter`);
     cy.contains('Status: Angemeldet als "workspace_admin"')
       .should('exist');
-    cy.contains('sample_workspace')
-      .click();
-    cy.url().should('eq', `${Cypress.config().baseUrl}/#/admin/1/files`);
-    cy.get('[data-cy="upload-files"]')
-      .should('be.enabled')
-      .get('[data-cy="delete-files"]')
-      .should('be.enabled');
   });
 
-  it('Should delete an admin by clicking on the name', () => {
+  it('should be possible to delete a workspace admin by row marking', () => {
```

Comment:
'Should' groß wenn es in allen anderen Testfällen groß geschrieben ist.

Reply:
hab ich angepasst.